packages/app-project/stores/User/UserPersonalization/UserPersonalization.js

```diff
@@ -69,6 +67,7 @@ const UserPersonalization = types
       },
 
       load() {
+        self.notifications.fetchAndSubscribe()
```

Comment:
This might fix a load of project tests that fire off a request to the Talk API as soon as you create a mock store.